src/Main.ts

```diff
@@ -362,7 +364,7 @@ export class Main {
         }
     }
 
-    public startTimer(name: string, labels: MetricsLabels = {}) {
+    public startTimer(name: string, labels: MetricsLabels = {}): TimerFunc {
```

Comment:
Nice! I should have introduced `TimerFunc`.

Reply:
:)